### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by bbaustin**
- Delete unneeded comma and space
- Add missing colons

### Description
Very small typo fixes! The added colons match with another paragraph in the same section

### CODE REVIEW

**Comment by dipikabh** [comment]
Thanks @bbaustin for opening the pull request. I'll take a look at the changes shortly.

File: files/en-us/web/css/gradient/conic-gradient/index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ background: conic-gradient(
 - {{CSSxRef("&lt;angle&gt;")}}
   - : Preceded by the `from` keyterm, and taking an angle as its value, defines the gradient rotation in clockwise direction.
 - `<position>`
-  - : Using the same length, order and keyterm values as the [background-position](/en-US/docs/Web/CSS/background-position) property, the position defines center of the gradient. If omitted, the default value is `center`, meaning the gradient will be centered, .
+  - : Using the same length, order and keyterm values as the [background-position](/en-US/docs/Web/CSS/background-position) property, the position defines center of the gradient. If omitted, the default value is `center`, meaning the gradient will be centered.
```
Comment (dipikabh) [inline files/en-us/web/css/gradient/conic-gradient/index.md:55]:
```suggestion
  - : Using the same length, order, and keyterm values as the [`background-position`](/en-US/docs/Web/CSS/background-position) property, the `position` value defines center of the gradient. If not specified, the value used for `position` by default is `center`, meaning the gradient will be centered.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Welcome to the MDN Web Docs project, @bbaustin and thank you for your contribution 🎉.

Your changes are perfect! I've added a few more edits to improve the general clarity of the paragraph.

File: files/en-us/web/css/gradient/conic-gradient/index.md (hunk 1 above)
Comment (bbaustin) [reply files/en-us/web/css/gradient/conic-gradient/index.md:55]: I love the Oxford comma! Nice changes 👍

**Comment by bbaustin** [comment]
> Your changes are perfect! I've added a few more edits to improve the general clarity of the paragraph.

I committed your changes! Let me know if that wasn't the right thing to do :)

**Review by dipikabh — APPROVED** [review]
Thanks for incorporating the updates!

**Pull request merged**